CONTRIBUTING.md

```diff
@@ -0,0 +1,170 @@
+
+##### version
+
+Use the existing `release-*` version. If there isn't an existing version set to `null`.
+
+##### realm
+
+Acceptable values for this field must be set as a sequence:
+
+```yaml
+realm:
+    - <acceptable value 1>
+    - <acceptable value 2>
+    # - ...
+```
+
+##### nominal resolution
+
+Choose the appropriate value for the resolution used:
+
+| Config resolution | Nominal Resolution |
+| -- | -- |
+| 1&deg; | 100 km |
+
+`<Add more as applicable>`
+
+These are sourced from [the CMIP6 controlled vocabulary](https://github.com/WCRP-CMIP/CMIP6_CVs/blob/main/CMIP6_nominal_resolution.json). If your resolution differs from those listed you will need to make a pull request to add it to this documentation and the QA checks.
+
+##### keywords
+
+We have a "controlled vocabulary of keywords to prevent a proliferation of synonyms that mean the same thing, and to make it easy to populate these fields:
+
+| Topic | Keywords (mutually exclusive) |
+|--|--|
+| `<topic>` | `<keywords>` |
+
+##### reference
+
+An appropriate scientific reference for the configuration. `<Give example of an appropriate scientific reference for this model>`
+
+##### license
+
+This is the license that will apply to the model outputs for an experiment. This should be set to the [SPDX identifier](https://spdx.org/licenses/) for [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) (`CC-BY-4.0`) to alleviate users from the burden of choosing a license, and to ensure model outputs have a permissive license for reuse to encourage open and shareable science.
+
+##### url
+
+This is a bit tricky. Ideally this should be a URL to the GitHub (or similar) repository of the configuration *for the experiment being run*. So if we include this and require it to be filled then it should either be the URL pointing at the branch being modified, or a placeholder. Either way it should include a comment that it should be updated to reference the experiment being run.
+
+##### model
+
+`<Give acceptable model names>`
+
+#### Configuration settings
+
+##### restart_period
+
+This is checked to make sure a shorter run time hasn't been set during testing and forgotten to set back to the proper value. As it is difficult to create a general heuristic the values have been hard-coded to those shown below:
+
+| Config resolution | `restart_period`|
+| -- | -- |
+
+`<This table needs to be filled in for the model in question>`
+
+##### restart_freq
+
+This governs how what model restart files are retained.
+
+The requirement is simply that a date-based frequency be used so that restarts are saved in a reliable manner. Typical values are `1YS` or `5YS` for 0.1&deg; models, and `5YS` to `20YS` for 1&deg;. See the [payu documentation](https://payu.readthedocs.io/en/latest/config.html#model) for a detailed description of the meaning of the time units in `restart_freq`.
```

Comment:
```suggestion
The requirement is simply that a date-based frequency be used so that restarts are saved in a reliable manner. A typical value is `20YS`. See the [payu documentation](https://payu.readthedocs.io/en/latest/config.html#model) for a detailed description of the meaning of the time units in `restart_freq`.
```